index.html

```diff
@@ -316,7 +316,7 @@ <h2>The <dfn>ThingDescription</dfn> type</h2>
           let res = await fetch('https://tds.mythings.biz/sensor11');
           // ... additional checks possible on res.headers
           let td = await res.json();
-          let thing = new ConsumedThing(td);
+          let thing = await WOT.consume(td);
```

Comment:
Although it is possible to use the constructor, it is more correct to use the factory method, indeed.